src/ontology/mondo-edit.obo

```diff
@@ -199463,15 +199461,15 @@ is_a: MONDO:0003847 {source="MESH:C564473/inferred"} ! Mendelian disease
 
 [Term]
 id: MONDO:0010378
-name: X-linked hereditary sensory and autonomic neuropathy with deafness
+name: X-linked hereditary sensory and autonomic neuropathy with hearing loss
 def: "This syndrome is characterized by the association of an axonal sensory and autonomic neuropathy with hearing loss." [Orphanet:139583]
 subset: gard_rare {source="GARD:0012731"}
 subset: ordo_disease {source="Orphanet:139583"}
 synonym: "auditory neuropathy, X-linked, 1, with peripheral sensory neuropathy" RELATED [OMIM:300614]
 synonym: "deafness, X-linked 5" RELATED [GARD:0012731, MONDO:Lexical, OMIM:300614]
 synonym: "DFNX5" RELATED ABBREVIATION [MONDO:Lexical, OMIM:300614]
 synonym: "X-linked auditory neuropathy with peripheral sensory neuropathy type 1" EXACT [Orphanet:139583]
-synonym: "X-linked hereditary sensory and autonomic neuropathy with deafness" EXACT CLINGEN_PREFERRED []
+synonym: "X-linked hereditary sensory and autonomic neuropathy with deafness" NARROW CLINGEN_PREFERRED [OMIM:300614]
```

Comment:
there are some duplicates. I manually fixed a few of these. I think you tried to run the command to combine these but it seems that a lot of them were missed for some reason. I can manually fix these if needed.

Reply:
I dont think there are duplicates - some synonyms were simply exact and narrow at the same time? Or did youmean something else?